### DISCUSSION TITLE
Betterfox userjs in Floorp uses the v115 version for regular Firefox not the ESR version.

### DISCUSSION THREAD

**Discussion by godwyn09** [Floorp-Projects/Floorp#538] [Feature Request]
Since the Floorp base is from ESR115, it is better to update its userjs to Betterfox ESR 115. I think it is okay to use the ESR version of Betterfox because the base version is based at 115ESR and not the normal v115 version that is used in current Floorp.

Here's the link for Betterfox ESR.
Link: https://github.com/yokoffing/Betterfox/tree/esr115

Here's the userjs in Betterfox ESR.
Link: https://raw.githubusercontent.com/yokoffing/Betterfox/esr115/user.js

This is the link to the userjs that Floorp is using right now.
Link: https://github.com/yokoffing/Betterfox/releases/tag/115.0

![12](https://github.com/Floorp-Projects/Floorp/assets/130208402/79627ebe-80d4-4da5-866c-3b9bb960444a)

**Comment by Ryosuke-Asano** [comment]
OK~
